eng/pipelines/templates/variables/vmr-build.yml

```diff
@@ -62,6 +62,12 @@ variables:
 
 - name: almaLinuxX64Rid
   value: almalinux.8-x64
+- name: alpinelinuxX64Rid
+  value: linux-musl-x64
+- name: alpinelinuxArmRid
+  value: linux-musl-arm
+- name: alpinelinuxArm64Rid
+  value: linux-musl-arm64
```

Comment:
Naming these `alpinelinux` may cause confusion in comparison to the other `alpineLatest` variable as far as which one to use. A better naming prefix might be `linuxMusl`.

Reply:
Updated.